src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -730,7 +730,7 @@ private void PopPeekedEnvelope(MessageEnvelope envelope)
             [EnumeratorCancellation] CancellationToken cancellationToken = default)
         {
             var enumerable = InternalReceiveNAsync(numberOfMessages, RemainingOrDefault, true, cancellationToken)
-                .ConfigureAwait(false).WithCancellation(cancellationToken);
+                .WithCancellation(cancellationToken);
```

Comment:
We don't need `.WithCancellation()` here, token is already passed into the method.